CoreUtilities/src/au/gov/asd/tac/constellation/utilities/tooltip/TooltipPane.java

```diff
@@ -20,29 +20,37 @@
 import javafx.scene.layout.Pane;
 
 /**
- *
+ * TooltipPane allows for a tooltip to be presented over the top of a standard Javafx
+ * AnchorPane. There can only be one tooltip active at any time.
+ * 
  * @author sirius
  */
 public class TooltipPane extends AnchorPane {
 
-    private Pane tooltipNode = null;
+    protected Pane tooltipNode = null;
```

Comment:
Since this is a field rather than a constant, can you change this back to private and then add a getter instead (can make getter protected). I assume this was only changed for unit testing purposes